generic/tests/netperf.py

```diff
@@ -95,28 +95,38 @@ def mtu_set(mtu):
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
 
-    try:
-        vm.wait_for_serial_login(timeout=login_timeout, restart_network=True).close()
-    except virt_vm.VMIPAddressMissingError:
-        pass
-
-    if len(params.get("nics", "").split()) > 1:
-        session = vm.wait_for_login(nic_index=1, timeout=login_timeout)
-    else:
-        session = vm.wait_for_login(timeout=login_timeout)
-
     config_cmds = params.get("config_cmds")
     if config_cmds:
         for config_cmd in config_cmds.split(","):
             cmd = params.get(config_cmd.strip())
+            session = vm.wait_for_serial_login(timeout=login_timeout)
             if cmd:
                 s, o = session.cmd_status_output(cmd)
-                if s:
+                logging.info(o)
+                if "querysettings" in cmd:
+                    if ".sys" in o:
+                        verifier_clear_cmd = "verifier /reset"
+                        status, output = session.cmd_status_output(verifier_clear_cmd)
+                        logging.info(output)
+                        if ".sys" in output:
+                            msg = "% does not work correctly" % verifier_clear_cmd
+                            test.error(msg)
```

Comment:
@xiagao Would you please help review this block? Thanks.

Reply:
I suggest to move vm reboot here, as it happen after 'verifier /reset' cmd.